### PR TITLE
feat(metrics): add country and region to flow events

### PR DESCRIPTION
**Pull request by philbooth**
Related to mozilla/fxa-activity-metrics#108.

We'll also need to update the import scripts and the Heka filter.

@mozilla/fxa-devs r?

### CODE REVIEW

File: server/lib/amplitude.js [hunk 1]
```diff
@@ -124,12 +123,11 @@ const transform = initialize(SERVICES, EVENTS, FUZZY_EVENTS);
 
 module.exports = receiveEvent;
 
-function receiveEvent (event, request, data) {
+function receiveEvent (event, request, data, location) {
```
Comment (vladikoff) [inline server/lib/amplitude.js]: nit: it's strange that location is not part of `data`, feels like we are just gonna keep adding arguments here?

File: server/lib/flow-event.js [hunk 2]
```diff
@@ -124,7 +126,7 @@ const metricsRequest = (req, metrics, requestReceivedTime) => {
       }
     }
 
-    amplitude(event, req, metrics);
+    amplitude(event, req, metrics, location);
```
Comment (vladikoff) [inline server/lib/flow-event.js]: same as https://github.com/mozilla/fxa-content-server/pull/6851#discussion_r244338400

Comment (philbooth) [reply server/lib/flow-event.js]: I was just trying to avoid either cloning or modifying the incoming `metrics` arg is all. I'll clone it instead.

Comment (vladikoff) [reply server/lib/flow-event.js]: Your choice!

**Pull request merged**